package.json

```diff
@@ -9,7 +9,7 @@
     "test": "jest",
     "start": "./node_modules/.bin/start-storybook -p 6006",
     "build": "npm run build:lib && npm run build:dist ",
-    "build:lib": "rimraf lib && ./node_modules/.bin/webpack --config webpack.config.js",
+    "build:lib": "rimraf lib && NODE_OPTIONS=--openssl-legacy-provider ./node_modules/.bin/webpack --config webpack.config.js",
```

Comment:
You've removed the `NODE_OPTIONS=--openssl-legacy-provider` and then you've added it again. If there's an issue with the node version in Vercel, we can update it.